src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -249,8 +249,14 @@ protected function whereRaw(Builder $query, $where)
     protected function whereBasic(Builder $query, $where)
     {
         $value = $this->parameter($where['value']);
+        $operator = $where['operator'];
 
-        return $this->wrap($where['column']).' '.$where['operator'].' '.$value;
+        // @see https://wiki.php.net/rfc/pdo_escape_placeholders
+        if (Str::contains(strtolower($operator), '?')) {
```

Comment:
We don't need `strtolower()`, right?

We could consider removing the `Str::contains()` check and just applying `str_replace()` all the time.